builders/interact/ethereum-api/precompiles/batch.md

```diff
@@ -0,0 +1,267 @@
+
+Try transferring the native token of your ContainerChain to two wallets of your choice via the batch precompile:
+
+1. Expand the batch contract under **Deployed Contracts**
+2. Expand the **batchAll** function
+3. For the **to** input, insert your addresses in the following format: `["INSERT_ADDRESS_1", "INSERT_ADDRESS_2"]`, where the first address corresponds to the first wallet of your choice and the second address corresponds to the second wallet of your choice
+4. For the **value** input, insert the amount you wish to transfer in Wei for each address. For example, `["1000000000000000000", "2000000000000000000"]` will transfer 1 native token to the first address and 2 native tokens to the second address
+5. For **callData**, insert `[]`. Call data is not relevant for simply transferring the native token
+6. For the **gasLimit** inputs, insert `[]`
+7. Press **transact**
+8. Press **Confirm** in the MetaMask extension to confirm the transaction
+
+![Send Batch Transfer](/images/builders/interact/ethereum-api/precompiles/batch/batch-4.webp)
+
+Once the transaction is complete, you can check both of the accounts' balances, either in MetaMask or in your ContainerChain's block explorer, a link to which can be found on the [Tanssi dApp](https://apps.tanssi.network/){target=\_blank}. Congratulations! You've now sent a batched transfer via the batch precompile.
+
+!!! note
+     Typically if you wanted to send the native currency to or through a contract, you would have to set the value within the overall transaction object and interact with a payable function. However, since the batch precompile interacts directly with Substrate code, this is not a typical Ethereum transaction and is thus not necessary.
+
+### Find a Contract Interaction's Call Data {: #find-a-contract-interactions-call-data }
+
+Visual interfaces like [Remix](/builders/interact/ethereum-api/dev-env/remix/){target=\_blank} and handy libraries like [Ethers.js](/builders/interact/ethereum-api/libraries/ethersjs/){target=\_blank} hide the way that Ethereum transactions interact with Solidity smart contracts. The name and input types of a function are hashed into a [function selector](https://docs.soliditylang.org/en/latest/abi-spec.html#function-selector-and-argument-encoding){target=\_blank} and the input data is encoded. These two pieces are then combined and sent as the transaction's call data. To send a subtransaction within a batch transaction, the sender needs to know its call data beforehand.
+
+Try finding a transaction's call data using Remix:
+
+1. Expand the `SimpleContract.sol` contract under **Deployed Contracts**
+2. Expand the **setMessage** function
+3. Enter the input of the function. For this example, **id** will be `1` and **message** will be `"tanssi"`
+4. Instead of sending the transaction, click the copy button next to the **transact** button to copy the call data
+
+![Transaction Call Data](/images/builders/interact/ethereum-api/precompiles/batch/batch-5.webp)
```

Comment:
There are 5 steps in the image, but only 4 steps listed here

Reply:
fixed ty